index.md

```diff
@@ -9,55 +9,192 @@ iOS and Android from a single codebase.
 
 {% include note.html content="_Flutter is an early-stage open-source project_. We hope you try Flutter out and send us [feedback](mailto:flutter-dev@googlegroups.com)." %}
 
-## New to Flutter
+## What is Flutter?
 
-See these topics to quickly start building cross-platform mobile apps with Flutter.
+Flutter provides an easy and productive way to build and deploy high-performance 
+mobile apps for Android and iOS. The goal is to deliver apps that feel natural 
+on different platforms, and we embrace differences like scrolling behavior, 
+overscroll behaviors, and more. 
 
- - [Set up your machine](/setup/) with Flutter and iOS/Android tools.
- - [Get started](getting-started) to run your first app on Android and iOS.
- - [A tour of the Flutter framework](widgets-intro)
- - [Bootstrap into Dart](/bootstrap-into-dart/) if you are new to the language
- for developing Flutter apps.
+![Shrine, iOS left and Android right](/images/whatisflutter/shrine_both.png)
 
-## Developer Guides
+This is an online marketplace app from the [Gallery](https://github.com/flutter/flutter/tree/master/examples/flutter_gallery/lib/demo), 
+a showcase of Flutter apps to download and run on a phone. Shrine has 
+high-quality scrolling images, interactive cards, buttons and dropdown lists, 
+and a shopping cart page. Run Shrine on an iPhone or Android phone; the UI 
+appearance and behavior are equivalent. Feel free to browse through the source 
```

Comment:
Yep, I wondered about that...

How about "Run Shrine on an iPhone or Android phone, then check
out the single codebase [here](https://github.com/flutter/flutter/tree/master/examples) 
in our github repository."